source/Octopus.Tentacle.Tests.Integration/Support/TentacleTestCaseSourceAttribute.cs

```diff
@@ -166,6 +172,11 @@ private IEnumerable<ITestCaseData> GetTestCasesFor(IMethodInfo method)
                             parms.Properties.Add(PropertyNames.Category, Net60ClientNet48Service);
                         }
                         #endif
+                        
+                        if (item is not TentacleConfigurationTestCase {Version: null} version)
```

Comment:
My IDE is saying "Local variable 'version' is never used". 

You can delete `version` and should still work.

Reply:
Same with `testCase` a few lines above.